monai/utils/type_conversion.py

```diff
@@ -241,7 +241,7 @@ def convert_data_type(
             (1.0, <class 'torch.Tensor'>, None)
 
     """
-    orig_type: Any
+    orig_type: type
```

Comment:
Should it be `torch.Tensor` or `np.ndarray`?

Thanks.

Reply:
this one could be any type, I couldn't give a more specific hint. this return value is not used frequently in our codebase, perhaps it's fine for now.